oaknational/Oak-Web-Application
# Feat - onboarding pages - LESQ-1022

## Description

Music year: {owa_music_year}

- Refactors feature flag into HOC
- Add's onboarding pages   
`/onboarding `   ` /onboarding/school-selection`.   `/onboarding/role-selection`
- `/onboarding` form style and and routing from url will be in another PR - wanted to get this in so it doesn't block other PR's
- `school-selection` has the school picker form from that was previously on `onboarding` -  this should work the same as before
    
## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2682--oak-web-application.netlify.thenational.academy/onboarding
You should be able to navigate to he new pages



## Checklist
